src/pages/iou/IOUModal.js

```diff
@@ -138,6 +179,33 @@ class IOUModal extends Component {
         this.setState({selectedCurrency});
     }
 
+    closeModal() {
+        Navigation.dismissModal();
+    }
```

Comment:
I can't where this is called. Is it unused?

Reply:
I think you were planning to all it on line 84, but It's probably not necessary.

Reply:
Yup, sorry!